subsys/usb/class/netusb/Kconfig

```diff
@@ -78,4 +78,41 @@ config USB_DEVICE_NETWORK_ECM_MAC
 
 endif # USB_DEVICE_NETWORK_ECM
 
+if USB_DEVICE_NETWORK_RNDIS
```

Comment:
Merge that patch with "usb: rndis: Add Remote NDIS protocol handling"

Reply:
OK